src/Microsoft.AspNet.Mvc.Core/Description/ApiDescriptionActionData.cs

```diff
@@ -4,13 +4,13 @@
 namespace Microsoft.AspNet.Mvc.Description
```

Comment:
IMO if we're changing this namespace we should completely get rid of the old one. **NO MERCY**